docs/zh/latest/getting-started.md

```diff
@@ -21,17 +21,33 @@ title: 快速入门指南
 #
 -->
 
-本指南旨在让大家入门 Apache APISIX，我们将配置一个对外提供公共 API 的服务，并由 API key 进行访问保护。
+## 概述
 
-另外，我们将以下面的 `echo` 端点为例，它将返回我们传递的参数。
+本文是 Apache APISIX 的快速入门指南。快速入门分为三个步骤：
 
-**Request**
+1. 通过[Docker](https://www.docker.com/) 和 [Docker Compose](https://docs.docker.com/compose/) 安装 Apache APISIX。
+1. 创建路由并绑定后端服务。
```

Comment:
```suggestion
2. 创建路由并绑定后端服务。
```

Reply:
Doucusarus will automatically change this '111....1'to a '123...n' order. I prefer leave it as it is. 

In case we have to add some more steps to it, then I would need to only add a '1. step description' to the document, rather than change all orders one by one.